.github/workflows/branch-to-staging.yml

```diff
@@ -3,10 +3,18 @@ name: Branch to Staging
 # Used to manually deploy a branch to staging
 
 on:
-  workflow_dispatch: # manually executed by a user
+  workflow_dispatch:
+    inputs:
+      jobs_to_run:
+        description: 'Select jobs to run (comma-separated): api,mllp-server,infra-api-lambdas,redeploy-api-gw,e2e-tests'
+        required: true
+        default: 'api,mllp-server,infra-api-lambdas,redeploy-api-gw,e2e-tests'
+        type: string
 
 jobs:
   api:
+    # can't use contains(github.event.inputs.jobs_to_run, 'api') because this would trigger on 'redeploy-api-gw' or 'infra-api-lambdas'
+    if: contains(format(',{0},', github.event.inputs.jobs_to_run), ',api,')
```

Comment:
🧙

Reply:
looks gross, but seems fine?

Reply:
I didn't know `format(..)` in this context, assuming you tried it somehow or got it from a good reference. Worst case we try it once this is merged.

Reply:
yep listed [here](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#functions). Not a lot of functions available, but nice that format is one of them